solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -421,6 +435,27 @@ contract LiquidityPoolV1Converter is LiquidityPoolConverter {
         return amount;
     }
 
+    /**
+      * @dev calculates the amount of each reserve token entitled for a given amount of pool tokens

```

Comment:
same comment here about `calculates` vs `returns`

Reply:
It appears in a few other places. Let's fix that once we merge both PRs into 0.6.17, because of this entanglement (of handling two PRs).

Reply:
Done